integration-libs/s4-service/order/components/cancel-service-order/cancel-service-order.component.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { OrderEntry } from '@spartacus/cart/base/root';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  RoutingService,
+} from '@spartacus/core';
+import { OrderDetailsService } from '@spartacus/order/components';
+import {
+  CancelServiceOrderFacade,
+  CancellationDetails,
+} from '@spartacus/s4-service/root';
+import { mergeMap, Observable, throwError } from 'rxjs';
```

Comment:
`major:` `mergeMap` operator should be imported from `rxjs/operators` module. Module `rxjs` has no exported member `mergeMap`